### PR TITLE
[MODINVOICE-449]-Adjustments cause blocker on invoices.

### PR DESCRIPTION
**Pull request by SinghAdes**
## Purpose
https://issues.folio.org/browse/MODINVOICE-449
## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

**Comment by SinghAdes** [comment]
image

File: acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-invoice-line-validation-with-adjustments.feature [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Feature: Check invoiceLine validation with  adjustments
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-invoice-line-validation-with-adjustments.feature:1]: Please add jira story name, and I think this tests should be renamed to something like this: check-invoice-validation-for-adjustments-not-fails-with-correct-adjustments

Comment (SerhiiNosko) [reply acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-invoice-line-validation-with-adjustments.feature:1]: or if we remain existing name - we need to add 2 scenarios, the first with failed validation, the second - with passed

Comment (SinghAdes) [reply acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-invoice-line-validation-with-adjustments.feature]: We removed the logic of validation from invoiceLine , so i dont think faliure will occur in any case. So renaming it to suggested name.

Comment (SerhiiNosko) [reply acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-invoice-line-validation-with-adjustments.feature:3]: okay, you added failure scenario, thanks

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**